server/models/Profile.js

```diff
@@ -0,0 +1,59 @@
+const mongoose = require('mongoose');
+
+const dayAvalSchema = new mongoose.Schema({
+    from: String,
+    to: String,
+    isAvailable: {
+        type: Boolean,
+        default: true
+    },
```

Comment:
Do you mind elaborating why we need this field? Is it so we can distinguish blocks of availability that have been booked?